src/main/java/uk/gov/companieshouse/api/strikeoffobjections/client/OracleQueryClient.java

```diff
@@ -22,4 +22,12 @@ public Long getCompanyActionCode(String companyNumber) {
 
         return response.getBody();
     }
+
+    public String getRequestedGaz2(String companyNumber) {
+        String getRequestedGaz2Url = String.format("%s/company/%s/gaz2-requested", oracleQueryApiUrl, companyNumber);
```

Comment:
could do with an INFO? log to say that we're calling the api, might help to help identify live issues